content/sensu-go/5.19/reference/agent.md

```diff
@@ -807,6 +809,32 @@ annotations:
 {{< /highlight >}}
 
 
+| assets-burst-limit   |      |
+--------------|------
+description   | Maximum amount of burst allowed in a rate interval when fetching assets. Added in [Sensu Go 5.19.3][55].
```

Comment:
Oh I see, it's here. It's probably best to remove the additional text from the help usage if its documented here!

Reply:
Yes, I agree. I made the updates...thank you!